front/src/applications/operationalStudies/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -96,14 +100,20 @@ const handleOperation = async ({
     }
     case 'update': {
       const trainrunIdToUpdate = createdTrainrun.get(trainrun.id) || trainrun.id;
+      const trainSchedule = await dispatch(
```

Comment:
For your information, later, we will have already fetched all the trainSchedules of the timetable in the parent component and we may pass the trainSchedules directly in props.

Reply:
My hope was that the automagic rtk-query cache would already contain the train schedule. But I haven't verified whether in practice it works that way.